Web/ForgeTrust.Runnable.Web.RazorDocs/wwwroot/css/app.css

```
@@ -278,6 +278,207 @@ body {
+
+.docs-outline-toggle:focus-visible {
+    outline: 2px solid #22d3ee;
+    outline-offset: -2px;
+}
+
+.docs-outline-toggle::after {
+    content: "";
+    width: 0.45rem;
+    height: 0.45rem;
+    flex: 0 0 auto;
+    border-right: 1.5px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)